src/auth/auth.module.ts

```diff
@@ -11,5 +11,6 @@ import * as REDIS_CONFIG from 'src/config/redis';
     TypeOrmModule.forFeature([User]),
```

Comment:
also why do you need user repo in auth.module, its better to import user service and use it's functions

Reply:
first of all, I don't have findOne method in my UsersService, and secondly, It requires more dependencies in case I use service instead of repo